docs/healpix.rst

```diff
@@ -0,0 +1,55 @@
+**************
+HEALPIX images
```

Comment:
The `healpix.rst` and HEALPIX high-level docstrings don't really say what method is used for reprojection.
I think it's bilinear interpolation in both directions?
Can you state this?